.github/workflows/containers.yml

```diff
@@ -0,0 +1,37 @@
+name: Containers
+
+on: [push, pull_request]
+
+jobs:
+  containers:
+    name: Buildah
```

Comment:
When using a strategy, it makes sense to a) first list the strategy, then b) include the values from the matrix in the name.

Reply:
Thanks for the tip, fixed in f297691.